pycaret/internal/plots/time_series.py

```diff
@@ -450,14 +450,36 @@ def plot_acf(
     data_kwargs: Optional[Dict] = None,
     fig_kwargs: Optional[Dict] = None,
 ) -> PlotReturnType:
-    """Plots the ACF on the data provided"""
-    fig, return_data_dict = None, None
-
+    """Plots the ACF on the data provided
+
+    Parameters
+    ----------
+    data : pd.Series
+        Data whose correlation plot needs to be plotted
+    fig_defaults : Dict[str, Any]
+        The defaults dictionary containing keys for "width" and "height" (mandatory)
+    model_name : Optional[str]
+        The name of the model, by default None
+    data_kwargs : Dict[str, Any]
+        A dictionary containing options keys for "nlags"
+    fig_kwargs : Dict[str, Any]
+        A dictionary containing options keys for "width" and/or "height"
+
+    Returns
+    -------
+    Tuple[go.Figure, Dict[str, Any]]
+        Returns back the plotly figure along with the correlation data.
+    """
     data_kwargs = data_kwargs or {}
     fig_kwargs = fig_kwargs or {}
 
     nlags = data_kwargs.get("nlags", None)
-    corr_array = acf(data, alpha=0.05, nlags=nlags)
+    name = "ACF"
+
+    subplots = make_subplots(rows=1, cols=1)
+    fig, acf_data = corr_subplot(
+        fig=subplots, data=data, col=1, row=1, name=name, plot="acf", nlags=nlags
```

Comment:
You do not have to pass name = name here since it defaults to "ACF". If you are changing it, then you can pass it.

Reply:
Any reason why you want to pass name = name here. The default is already set to ACF for this plot internally.